cmd/snap-update-ns/utils.go

```diff
@@ -481,13 +489,19 @@ func planWritableMimic(dir, neededBy string) ([]*Change, error) {
 			// umount2(2) system call.
 			Name: dir, Dir: safeKeepingDir, Options: []string{"rbind"}},
 	})
+
 	// Mount tmpfs over the original directory, hiding its contents.
+	// The mounted tmpfs will mimic the mode and ownership of the original
+	// directory.
 	changes = append(changes, &Change{
 		Action: Mount, Entry: osutil.MountEntry{
 			Name: "tmpfs", Dir: dir, Type: "tmpfs",
 			Options: []string{
 				osutil.XSnapdSynthetic(),
 				osutil.XSnapdNeededBy(neededBy),
+				fmt.Sprintf("mode=%#o", sb.Mode&07777),
+				fmt.Sprintf("uid=%d", sb.Uid),
```

Comment:
`%u`?

Reply:
There is no such a thing: https://golang.org/pkg/fmt/